modules/ROOT/edits/datagraph-settings.adoc

```diff
@@ -0,0 +1,23 @@
+= Anypoint DataGraph Administration Settings
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+On the DataGraph Administration settings page, you can select environments, view and edit endpoints, configure policies, and configure client providers.
+
+image::datagraph-settings.png[The illustration shows the Anypoint DataGraph Administration page]
+
+<1> *Switch Environment* control. Click the control to see a list of environments you have added to your organization. Select an environment to switch.
+
+<1> *Consumer endpoint* edit options. To edit the current endpoint, click the *Edit consumer endpoint* icon. To revert to the system-generated endpoint, click the *Revert* icon.
+
+<1> *DataGraph Policies* configuration options. You can choose to restrict requests to your unified schema from registered client applications only, set SLA-based rate limiting for requests, or allow requests from anyone with access to the unified schema's endpoint.
+
+<1> *DataGraph external client providers* control. To see a list of external client providers, previously configured by your organization administrators, click the control. Select a provider from the list. To run queries in Anypoint DataGraph, you'll need the proper credentials from the external client provider.
```

Comment:
To avoid future tense:
```suggestion
<1> *DataGraph external client providers* control. To see a list of external client providers previously configured by your organization administrators, click the control. Select a provider from the list. To run queries in Anypoint DataGraph, obtain proper credentials from the external client provider.
```